Dockerfile

```diff
@@ -17,8 +17,7 @@ RUN cargo build --example empty --release && \
     rm -rf ./target/release/.fingerprint/krustlet-*
 
 # Build real binaries now
-RUN rm ./src/main.rs ./src/lib.rs
 COPY ./src ./src
 
 RUN cargo build --release
-CMD ["/usr/src/krustlet/target/release/krustlet"]
\ No newline at end of file
+CMD ["/usr/src/krustlet/target/release/krustlet"]
```

Comment:
this might have to change to `krustlet-wasi` or to `krustlet-wascc`. krustlet no longer is a single binary.

@oftaylor any opinions what we should do here?